### PR TITLE
Improve playbook script and documention

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:
* Clean up `utils/ansible_playbook_to_role.py`
* Add docs for `utils/ansible_playbook_to_role.py`

#### Rationale:

Follow up to #11495

#### Review Hints:

Follow the steps in the new docs.

### CODE REVIEW

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -350,3 +350,19 @@ if(PYTHON_VERSION_MAJOR GREATER 2 AND PYTHON_VERSION_MINOR GREATER 6)
...
+if(PYTHON_VERSION_MAJOR GREATER 2 AND PY_GITHUB)
+    add_test(
+        NAME "utils-ansible-playbook-to-role-help"
+        COMMAND env "PYTHONPATH=$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/utils/ansible_playbook_to_role.py" "--help"
+    )
+    set_tests_properties("utils-ansible-playbook-to-role-help" PROPERTIES LABELS quick)
+    if(SSG_PRODUCT_RHEL9)
+        file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/roles")
+        add_test(
+            NAME "utils-ansible-playbook-to-role-rhel9"
```
Comment (jan-cerny) [inline tests/CMakeLists.txt:364]: Is there any GitHub CI job that runs this test?

**Review by jan-cerny — COMMENTED** [review]
I followed the instructions in the doc and the script works fine. I also run the new tests locally and they passed.

File: tests/CMakeLists.txt (hunk 1 above)
Comment (Mab879) [reply tests/CMakeLists.txt:364]: I will update the requirements to make the tests run.

**Review by jan-cerny — APPROVED** [review]
I have seen both tests running in the Build Test on Fedora Latest job.

**Pull request merged**